bridge/ffi/src/errors.rs

```diff
@@ -0,0 +1,29 @@
+use dsnp_graph_config::errors::DsnpGraphError;
+
+// Opaque handle for DsnpGraphError
+pub struct DsnpGraphErrorHandle {
+	// to make errors opaque this field must be private
+	// it enables to create a pointer to the error
+	error: *mut DsnpGraphError,
```

Comment:
we can expose underlying type transparently due to error types, so keeping it opaque as consumer would call provided functions to get error code and message